src/formatDate.js

```diff
@@ -8,7 +8,30 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.pop();
+  const toSeparator = toFormat.pop();
```

Comment:
Similar to the previous comment, using `pop` on the `toFormat` array will modify the original array. It's best to work with a copy to prevent unintended mutations.